trainingsanleitungen/scratch/scratch-fang-mich-v3.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: sushi
+title: Scratch Fang-Mich
+description: In diesem Spiel musst du versuchen, dem Haifisch zu entkommen.
+---
+
+# Scratch Fang-Mich
+
+In diesem Spiel bist du ein Fisch, der dem großen Haifisch entkommen muss. Schaffst du es?
+
+## Bühne und Figuren anlegen
+
+1. ![Bühnenbild auswählen](scratch-fang-mich-v3/01-background.png){: .right}
+Als erstes legst du fest, wie dein Spielfeld aussehen soll. Wir brauchen zuerst das Aquarium, in dem die Fische schwimmen. Wähle als erstes rechts unten unter *Bühnenbild wählen* ein Bühnenbild aus, zum Beispiel ein Underwater 1.
+
+2. ![Figur löschen](scratch-fang-mich-v3/02-delete-scratchy.png){: .right}
+Als nächstes lösche die Figur Scratchy mit dem Namen *Sprite 1* indem du mit der rechten Maustaste darauf klickst. 
+Im angezeigten Menü kannst du Scratchy löschen.
+
+3. ![Figuren hinzufügen](scratch-fang-mich-v3/03-fish.png){: .right}
+Jetzt brauchen wir einen Haifisch sowie einen Fisch, mit dem wir dem Haifisch entkommen wollen. Klicke dazu auf *Figur wählen* und füge einen Fisch sowie einen Haifisch dazu. Natürlich können es auch andere Figuren sein, zum Beispiel ein Käfer, der dem Vogel davonläuft oder eine Person die dem Löwen davon läuft.
+
+4. ![Fisch gruppieren](scratch-fang-mich-v3/04-to-group.png){: .right}
+Damit der kleine Fisch auch kleiner ist als der große Haifisch, müssen wir den Fisch verkleinern. Davor müssen wir ihn aber noch *gruppieren*, sodass im nächsten Schritt der ganze Fisch kleiner wird, nicht nur die Schwanzflosse. Wähle dazu den Fisch bei den Figuren aus, sodass er blau umrandet ist. Klicke anschließend auf den Tab *Kostüme*. Führe die Maus auf die karrierte Fläche und drücke die linke Maustaste. Halte diese gedrückt und bewege deine Maus nun so, dass sich der ganze Fisch innerhalb der grauen Linien befindet. Lass los und führe deine Maus nach oben, wo du rechts neben den beiden blauben Pfeilen ein Symbol siehst. Drücke darauf und der Fisch ist nun gruppiert.
+
+5. ![Fisch verkleinern](scratch-fang-mich-v3/05-resize.png){: .right}
+Nun kannst du den Fisch verkleinern, indem du ihn an den Ecken kleiner ziehst. *Tipp:* Wenn du den Fisch nur kleiner machen möchstest ohne ihn zu viel zu verändern, halte die Shift-Taste gedrückt, während du in kleiner ziehst.
+
+7. ![Figuren benennen](scratch-fang-mich-v3/07-rename.png){: .right}
+Damit du später die Figuren leichter verwenden kannst, gib ihnen Namen wie *Hai* und *Fisch*. Du kannst den Namen von Figuren ändern, indem du unter der weißen Leinwand rechts neben dem Wort Figur auf das Namensfeld klickst und den neuen Namen eingibst.
+
+## Fisch bewegen
+
+1. ![Fisch bewegen](scratch-fang-mich-v3/08-move-fish.png){: .right}
+Damit du den Fisch bewegen kannst, musst er nach links und rechts sowie oben und unten bewegt werden können.<br/><br/>
+  • Wähle zuerst den Fisch aus, damit er blau umrandet ist.<br/>
```

Comment:
Ändere den Aufzählungspunkt auf einen normalen Stern (*), war in der ursprünglichen Anleitung schon falsch. Diese Anmerkung gilt für das ganze Dokument.